invokeai/app/services/graph.py

```diff
@@ -805,7 +807,7 @@ class Config:
             ]
         }
 
-    def next(self) -> BaseInvocation | None:
+    def next(self) -> Union[BaseInvocation, None]:
```

Comment:
This should be functionally the same as the prior syntax?

Reply:
Functionally identical, but the Union form is needed by Python 3.9 which is still the highest python you can get on Ubuntu 20.04 LTS systems. Maybe there's a future compatibility module I can load?